proto/build.rs

```diff
@@ -98,7 +90,7 @@ fn main() -> Result<()> {
                 "vendor/cosmos/tx/v1beta1/tx.proto",
                 "vendor/go-header/p2p/pb/header_request.proto",
             ],
-            &["vendor"],
+            &["vendor", "vendor/nmt"],
```

Comment:
is this a bug (or inconsistency) in their change?

Reply:
Inconsistency from our point of view. Their code is correct. I had two options:

1. `proto/vendor/pb/proof.proto`
2. `proto/vendor/nmt/pb/proof.proto`

I choose 2 to make it consistent to our directory structure.

Reply:
yeah, I think they just haven't noticed, but we don't have to bother them